### DISCUSSION TITLE
[ FF8 ] OpenGL green bug

### DISCUSSION THREAD

**Discussion by shikulja** [julianxhokaxhiu/FFNx#191] [General]
intel hd630 opengl green bug
![Аннотация 2020-06-03 215318](https://user-images.githubusercontent.com/3386359/83677336-af46ca00-a5e4-11ea-8ce5-ad45f5e968c6.png)

and release the latest canary build please

**Comment by julianxhokaxhiu** [comment]
Please post your `FFNx.log` file and before doing that, enable `renderer_debug = yes`. I'm not able to replicate here ( dual GPU Intel HD 620 / Nvidia MX 130 ).

Also make sure you have the most up-to-date drivers for your Intel GPU.

**Comment by shikulja** [comment]
[FFNx.log](https://github.com/julianxhokaxhiu/FFNx/files/4740033/FFNx.log)


![Аннотация 2020-06-06 153957](https://user-images.githubusercontent.com/3386359/83944419-037fc300-a80c-11ea-9245-01277b5aed8e.png)

**Comment by julianxhokaxhiu** [comment]
You do have this hext in your folder:
```
[00000000] TRACE: Applied Hext patch: hext\ff8\en_nv\FF8_1.2NV.CD.fix.FBSA.txt
```
Try to remove that and please use an original Disk. It may be you're patching something that is causing that. Make sure you do use a legit copy before opening bug reports. Thank you.
